public/index.html

```diff
@@ -7,36 +7,234 @@
+
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/2.png" alt="">
+        <h2>Multi-Post
+            Stories</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/3.png" alt="">
+        <h2>Tonic</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/4.png" alt="">
+        <h2>Multi-Post
+            Stories</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    <section class="last-section">
+        <h1>About Myself</h1>
```

Comment:
- [ ] **[OPTIONAL]** Please consider using `h2` and `h3` instead of `h1`for these different title. Each page should have only one `h1`. This will improve highly the accessibility of your website 👍 💯